src/server/network.ts

```diff
@@ -127,6 +130,10 @@ export class Request {
     return this._headers;
   }
 
+  headerValue(name: string): string | undefined {
```

Comment:
I guess this is fine today because the server Request is not public anymore?

Reply:
Yes, exactly.